versioned_docs/version-8.7/self-managed/identity/application-user-group-role-management/identity-application-user-group-role-management-overview.md

```diff
@@ -0,0 +1,17 @@
+---
+id: identity-application-user-group-role-management-overview
+title: "Users, applications, groups, and roles overview"
+sidebar_label: "Overview"
+description: "Learn more about how Identity makes use of users, groups, and roles"
+---
+
+Identity allows you to manage:
+
+- **Users** that represent Camunda 8 users (e.g., `Joe Doe`).
```

Comment:
```suggestion
- **Users** that represent humans interacting with Camunda 8 UIs (e.g., `Joe Doe`).
```

More similar to the second bullet point and thus maybe easier to tell what the difference is

Reply:
Good point. Added